arangod/Sharding/ShardingInfo.cpp

```diff
@@ -425,11 +430,20 @@ void ShardingInfo::setWriteConcernAndReplicationFactor(size_t writeConcern, size
 
 bool ShardingInfo::isSatellite() const { return _replicationFactor == 0; }
 
+std::pair<bool, bool> ShardingInfo::markAsSatellite() {
```

Comment:
can we call this "makeSatellite" ?
It will not only declare it as one, but also modify all other parameters under the hood.

Reply:
renamed.